supabase/client.py

```diff
@@ -61,7 +59,7 @@ def __init__(
         #     supabase_key=self.supabase_key,
```

Comment:
Is this comment resolved too?

Reply:
@j0

Reply:
Hey nope I don't think it is, we need to implement realitme in order for it to be resolved